vpdq/python/vpdq.pyx

```diff
@@ -78,14 +76,14 @@ def hash_to_hex(hash_value: "Hash256"):
     Returns:
         str: hex str of hash
     """
-    return hashToString(hash_value)
+    return str(hashToString(hash_value), "utf-8")
 
 
 def str_to_hash(str_hash: str):
     return fromStringOrDie(str(str_hash).encode("utf-8"))
 
 
-def hamming_distance(hash1: "Hash256", hash2: "Hash256"):
```

Comment:
(unrelated to diff): Why is this function exposed? Should we instead direct people to the PDQ bindings instead?

Reply:
Because the PDQ binding does not provide a hamming distance function from the C++ implementation. I believe having this function will increase performance. The translation and calculation happen in PDQ_util instead of the binding.